lib/action.rb

```diff
@@ -39,11 +39,12 @@ def version_file_changed?(pull_number)
   end
 
   def version_increased?(branch_name:, trunk_name: 'master')
```

Comment:
I think this would be misleading if still called this. As it can return true when file is not found. I think a simple rename would improve readability. Something like `version_increased_or_version_file_not_found?`

Another option is to create another method that JUST checks the version file's existence, and the first thing the GitHub Action's main method does is call this method.

Currently, I think it's a bit hacky and `version_increased?`'s job should be checking just that, not also checking that the version file exists.

Reply:
The business rule of `version_increased?` is that if the file has renamed, that constitutes a version increase (since the module name is likely also different). That's why I didn't rename it.